### DISCUSSION TITLE
Web app editor doesn't load the image from drive

### DISCUSSION THREAD

**Discussion by Bouh** [4ian/GDevelop#2331] [Enhancement ideas (and their technical discussions)]
## Describe the bug
On the web app :
https://editor.gdevelop-app.com/
When you load a game coming from PC and open it on web app with Gdrive, the images are not loaded.

![image](https://user-images.githubusercontent.com/1670670/71525697-3e7fe100-28d3-11ea-9ffc-dd3b84fa78b5.png)

## To Reproduce
Open this project :
https://drive.google.com/drive/folders/1ASSFgxaHgYv9mA-b6KxZ6tjSDFSAgfC7?usp=sharing

## Other details
Only web app

**Comment by 4ian** [comment]
Yep I'm not sure exactly how to approach this. 
The web-app deals with URLs (and export download/store files in the resulting output zip file). The desktop app deals with (relative) paths on the local file system.

So in a way, projects are somewhat incompatible between web-app and desktop-app because of this. There should be a way to do web-app => desktop-app (basically, download the required files) and desktop-app => web-app (not sure how) or find another way to store resources that is common.

**Comment by Bouh** [comment]
Use the relative path everywhere in the game.json.
Store the absolute path to the game project in the game.json.

Reconstruct the path with or without the beginning of the absolute path at the resource function loading.
And according to the platform use reconstruct absolut path or use relative.

- If it's the web platform use the url to the game.json and add it relative path.
- If it's native app use the absolute path and att it relative path.
